tests/utils/test_github_contributors.py

```diff
@@ -0,0 +1,83 @@
+import pandas as pd
+from utils.github_contributors import contributors
+
+# Customize defaults for given_request_to
+__MOCK_REQUEST_DEFAULTS = {
+    "dir": "github",
+    "headers": {"Authorization": "Bearer token"}
+}
+
+GITHUB_GRAPHQL_URL = "https://api.github.com/graphql"
+
+
+def test_contributors(responses, given_request_to, mock_snowflake, load_dataset):
+    # GIVEN: repo query returns two pages of results
+    given_request_to(GITHUB_GRAPHQL_URL, "repo.page.1.json", method="POST")
+    given_request_to(GITHUB_GRAPHQL_URL, "repo.page.2.json", method="POST")
+    given_request_to(GITHUB_GRAPHQL_URL, "mattermost-server.page.1.json", method="POST")
+    given_request_to(GITHUB_GRAPHQL_URL, "mattermost-server.page.2.json", method="POST")
+    given_request_to(GITHUB_GRAPHQL_URL, "docs.page.1.json", method="POST")
+    given_request_to(GITHUB_GRAPHQL_URL, "focalboard.page.1.json", method="POST")
+
+    # GIVEN: mock snowflake connection
+    _, mock_connection, _, mock_to_sql = mock_snowflake("utils.github_contributors")
+
+    # WHEN: request to load contributors
+    contributors()
+
+    # THEN: expect github's graphql url to have been called for getting repose
```

Comment:
repose?

Reply:
Oops, meant `repos`. It's a typo, fixed now!.